src/scenes/ObservationsRouter/biomass/BiomassMeasurementMapView.tsx

```diff
@@ -32,21 +30,19 @@ export default function BiomassMeasurementMapView({
 }: BiomassMeasurementMapViewProps): JSX.Element {
   const dispatch = useAppDispatch();
   const [requestId, setRequestId] = useState<string>('');
-  const defaultTimeZone = useDefaultTimeZone();
 
   const observationHistory = useAppSelector((state) => selectPlantingSiteHistory(state, requestId));
 
   const [plantingSiteHistory, setPlantingSiteHistory] = useState<PlantingSiteHistory>();
 
   const observations: Observation[] | undefined = useAppSelector((state) =>
-    selectPlantingSiteObservations(state, selectedPlantingSite.id)
+    selectPlantingSiteAdHocObservations(state, selectedPlantingSite.id)
   );
 
   const observationsDates = useMemo(() => {
     const uniqueDates: Set<string> = new Set();
     observationsResults?.forEach((obs) => {
-      const timeZone = selectedPlantingSite?.timeZone ?? defaultTimeZone.get().id;
-      const dateToUse = obs.completedTime ? getDateDisplayValue(obs.completedTime, timeZone) : obs.startDate;
+      const dateToUse = obs.completedTime ? obs.completedTime : obs.startDate;
```

Comment:
NIT this can probably be updated now to `obs.completedTime || obs.startDate`
(same as other instance below too)